library/src/main/java/com/datatorrent/lib/io/fs/FileSplitterInput.java

```diff
@@ -452,7 +447,7 @@ private void scan(Path filePath, Path rootPath, Map<String, Long> lastModifiedTi
           }
         }
       } catch (FileNotFoundException fnf) {
-        LOG.warn("Failed to list directory {}", filePath, fnf);
+        LOG.error("Failed to list directory {}", filePath, fnf);
```

Comment:
The exception should be propagated.
Don't know why this was just logged before. The scanner is provided with a list of input folders so this assumes that it is fine if the scanner fails to open some of them. It is better to highlight these issues so propagating seems better.